src/core/logic.py

```diff
@@ -1,12 +1,12 @@
-from __builtin__ import any as string_any
+from builtins import any as string_any
```

Comment:
Could have just assigned `any` to `string_any` like thus

```string_any = any```

Then there's more than one way of skinning a cat

Reply:
Why would you do this, though?

Reply:
Why wouldn't you? Anyway are you talking about renaming `any` to `any_string`, or something else?

Reply:
Yeah, sorry, I wasn't paying attention to the `from builtins` part of the import - didn't realise this was the built-in `any` function. It is a bit odd renaming a Python though.

Reply:
I'm not sure why they bother either, but it's there. Who knows what was going through the developers mind at the time.